example-project/relayer-engine-config/common.json

```diff
@@ -3,8 +3,10 @@
   "storeType": "Redis",
   "readinessPort": 2000,
   "numGuardians": 1,
-  "redisPort": 6379,
-  "redisHost": "localhost",
+  "redis": {
```

Comment:
What about using a connection string format? Would be flexible to change if it is needed

Reply:
tried :(, didn't work for some values.